### ISSUE TITLE
Conflict between two plot instances

### ISSUE THREAD

**Issue by plankter** [flekschas/regl-scatterplot#17]
I have two `regl-scatterplot` instances in different components. When selection is performed on one of the plots (doesn't matter if selection is done via lasso tool or without holding Shift button), then second plot throws the following error during zooming/panning:

```js
regl-scatterplot.esm.js?6a7f:400 Uncaught TypeError: Cannot read property '0' of null
    at transformMat4 (regl-scatterplot.esm.js?6a7f:400)
    at getScatterGlPos (regl-scatterplot.esm.js?6a7f:2639)
    at raycast (regl-scatterplot.esm.js?6a7f:2645)
    at mouseMoveHandler (regl-scatterplot.esm.js?6a7f:2818)
```

UPDATE: the issue occurs only if `backgroundImage` property is set for the plot that throws the exception:

```js

const regl = this.scatterplot.get("regl");
const img = new Image();
img.crossOrigin = "";
img.src = this.channelStackImage as any;
img.onload = () => {
    this.scatterplot.set({
       backgroundImage: regl.texture(img),
    });
};

```

**Comment by flekschas** [comment]
I can't reproduce this after having fixed #16. Could you test this again with the latest master? Simply run `npm start` and go to `http://localhost:8080/two-instances.html`

**Comment by flekschas** [comment]
Closing given https://github.com/flekschas/regl-scatterplot/issues/16#issuecomment-638316251
